kokoro/pypi.sh

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Code under repo is checked out to this directory.
+cd "${KOKORO_ARTIFACTS_DIR}/github/dataflux-pytorch"
+
+function setup_virtual_envs() {
+    sudo apt-get -y update
+
+    echo Setting up Python virtual environment.
+    sudo apt install -y python3.8-venv
+    python3 -m venv venv
+    source venv/bin/activate
+}
+
+function run_git_commands() {
+    echo Setting git permissions.
+    git config --global --add safe.directory "*" 
+
+    echo Installing git submodules.
+    git submodule init
+
+    echo Updating git submodules.
+    git submodule update
+}
+
+function install_requirements() {
+    echo Installing python3-pip.
+    sudo apt-get -y install python3-pip
+
+    echo Installing project from pypi
+    pip install gcs-torch-dataflux
+}
+
+function run_integration_tests(){
```

Comment:
Could you add a comment here clarifying that the intention is to run the tests from PyPi? That way someone doesn't come along later and accidentally update it to run the local version